tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "jsx": "react",
     "baseUrl": "./src",
+    "esModuleInterop": true,
```

Comment:
Why do we need to turn this on?

Reply:
React wasn't importing properly before and VS code's suggestion was this...